src/lib/RpcApi.ts

```diff
@@ -114,11 +130,25 @@ export default class RpcApi {
                     hasRpcState: !!this._staticStore.rpcState,
                     hasRequest: !!this._staticStore.request,
                 });
-                this._router.push({name: request});
+                this.routerReplace(request);
             });
         }
     }
 
+    private _parseUrlParams(query: string) {
+        const params: {[key: string]: string} = {};
+        if (!query) return params;
+        const keyValues = query.substr(1).replace(/\+/g, ' ').split('&')
+            .map((keyValueString) => keyValueString.split('='));
```

Comment:
You could use URLSearchParams here.
```suggestion\
           const keyValues = new URLSearchParams(query);
```

Reply:
`URLSearchParams` decodes the params differently from what vue-router does (https://github.com/vuejs/vue-router/blob/dev/src/util/query.js#L65), which is exactly the problem I created this function for. Otherwise it would have been easy :laughing:.

Reply:
I see :D